AODN/AODN-WAVE-NRT/ARDC_API_NRT/ardc_nrt/config/sofar/template_nsw-dpe.json

```diff
@@ -166,8 +166,8 @@
   },
   "Conventions": "CF-1.6",
   "abstract": "Near real-time in-situ wave data from shallow coastal waters (<35 m depth) along the NSW coast collected by temporary wave buoys deployed on moorings.",
-  "acknowledgement": "Any users of data are required to clearly acknowledge the source of the material in the format: \\\"NSW nearshore wave buoy data were collected, processed and provided by the NSW Department of Planning and Environment Nearshore Wave Data Program\\\". For more information visit: https://datasets.seed.nsw.gov.au/dataset/nsw-nearshore-wave-buoy-parameter-time-series-data-completed-deployments. This dataset is an output from the Catching Oz Waves project, which received investment (https://doi.org/10.47486/DP748) from the Australian Research Data Commons (ARDC). The ARDC is funded by the National Collaborative Research Infrastructure Strategy (NCRIS). Data was sourced from Australia\\’s Integrated Marine Observing System (IMOS) - IMOS is enabled by the National Collaborative Research Infrastructure Strategy (NCRIS).\\",
-  "citation": "The citation to be used in publications using the dataset should follow the format: \"Department of Planning and Environment (DPE), New South Wales Government [year-of-data-downloaded], Wave buoys Observations - Australia - near real-time, Downloaded from [url] on [date-of-download].\"",
+  "acknowledgement": "Any users of data are required to clearly acknowledge the source of the material in the format: \\\"NSW wave data sourced from IMOS were collected, processed and provided by the NSW Nearshore Wave Data Program operated by the NSW Department of Climate Change, Energy, the Environment and Water\\\". For more information visit: https://datasets.seed.nsw.gov.au/dataset/nsw-nearshore-wave-buoy-parameter-time-series-data-completed-deployments. This dataset is an output from the Catching Oz Waves project, which received investment (https://doi.org/10.47486/DP748) from the Australian Research Data Commons (ARDC). The ARDC is funded by the National Collaborative Research Infrastructure Strategy (NCRIS). Data was sourced from Australia\\’s Integrated Marine Observing System (IMOS) - IMOS is enabled by the National Collaborative Research Infrastructure Strategy (NCRIS).\\\"",
+  "citation": "The citation to be used in publications using the dataset should follow the format: \"Department of Climate Change, Energy, the Environment and Water (DCCEEW), New South Wales Government [year-of-data-downloaded], Wave Buoys Observations - Australia - delayed (National Wave Archive), Downloaded from [url] on [date-of-download].\"",
```

Comment:
Sorry I made a mistake in the name of the collection which should be " Wave Buoys Observations - Australia - near real-time" instead of "Wave Buoys Observations - Australia - delayed (National Wave Archive)"